rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/00-fec3-sriov-node-policy.yaml

```diff
@@ -86,3 +86,24 @@ spec:
     deviceID: "159b"
     pfNames:
       - ens4f0#8-9
+---
+###############################################################################
+# netdevice for all core networks
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: core-network-devices
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  nodeSelector:
+    feature.node.kubernetes.io/open5gs: "true"
```

Comment:
L100 typically nodeselectors we utilize location based constructs, say for instance two different sites have the same nic PF and nic.  labeling the node with that location based construct is more granular and telling than a feature.

Reply:
We could have something like that:

```
nodeSelector:
  zone.node.kubernetes.io: "core"
```